installers/legacy-install.sh

```diff
@@ -125,10 +129,20 @@ x86_64)
 esac
 
 # Determine the tmp directory.
-if [ -z "$TMPDIR" ]; then
-  TMPDIR="/tmp"
-fi
+set_tempdir () {
+  if type mktemp > /dev/null; then
+    TMPDIR=`mktemp -d`
+  else
+    TMPDIR="${TMPDIR:-/tmp}/state-installer.$$"
+    # clean-up previous temp dir
+    rm -rf $tdir
+    mkdir -p $TMPDIR
+  fi
 
+}
+set_tempdir
```

Comment:
This doesn't seem relevant to this PR. Prefer we don't touch anything we don't need to.

Reply:
Okay, I can remove that.  I noticed that this was a block that wasn't unified between the installers.  But I can remove it.

Reply:
I've got nothing against it other than it could create some new bugs and I'd rather minimize the odds of that happening. This'll move into the go installer soon anyway.